e2e/features/wallet/Wallet.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Wallet
+  Background: Add sender and recipient accounts
+    Given I click on a button with text "CustomNode"
+    And I wait for "1 seconds"
+    Given I click on text "Add network"
+    Then I should see "dialogAddNetwork" modal
+    And button with text "Add network" should be disabled
+    Given I type "local_service" in "name"
+    And I type "http://127.0.0.1:9901" in "serviceUrl"
+    And I type "ws://127.0.0.1:9901" in "wsServiceUrl"
+    And I wait for "1 seconds"
+    And I click on a button with text "Add network"
+    Then I should see 'Custom network added "local_service"'
+    And I wait for "1 seconds"
+    Given I click on an element with testId "selected-menu-item"
+    And I click on exact text "local_service"
+    Then I go to page "wallet"
+    And I click on exact text "Add account"
+    Given I wait for "1 seconds"
+    Given I add an account with passphrase "fabric vivid advance supreme custom rail owner shrug amazing bird toast add" password "Password@1" name "recipient_0" custom derivation path "m/44'/134'/0'"
+    And I wait for "1 seconds"
+    Given I click on an element with class "account-management-dropdown"
+    And I click on text "Add new account"
+    Given I add an account with passphrase "attract squeeze option inflict dynamic end evoke love proof among random blanket table pumpkin general impose access toast undo extend fun employ agree dash" password "Password@1" name "genesis_70" custom derivation path "m/44'/134'/70'"
+    And I wait for "1 seconds"
+    And I go to page "wallet"
+    And I wait for "1 seconds"
+
+  Scenario: Send tokens
+    Given I click on a button with exact text "Send"
+    Then I should see send form details
+    And I should see "lisk_mainchain" should be selected in "To application" dropdown
+    And I should see "Lisk" should be selected in "Token" dropdown
+    Then button with text "Continue to summary" should be disabled
+    When I type "10" in "amount"
+    When I type "lsk9s5g8f7v6evazu2z6zo6oygzyz4skj7s64fst6" in "recipient"
```

Comment:
Add message with 64 chars..? we recently found a bug with request token where it was not supporting 64 char length.. also include special chars as part of message.

Reply:
Fixed